### DISCUSSION TITLE
Ability to set Spatial audio settings with shortcut

### DISCUSSION THREAD

**Discussion by flowerdealer** [Belphemur/SoundSwitch#1160] [FAQ]
Windows constantly disables spatial audio (such as Sonic/Dolby Atmos, etc), and you have to enable it manually. It would be great if you could change it or set it with a shortcut using this app.

**Comment by Belphemur** [comment]
Hello,
Already have a issue for it:
#410

For now, it's not planned, I don't have easy access to that API.
